### PR TITLE
Confiscation tracking

### PR DESCRIPTION
**Pull request by sqrrm**

### CODE REVIEW

File: core/src/main/java/bisq/core/dao/governance/param/Param.java [hunk 1]
```diff
@@ -150,6 +150,14 @@
         this(defaultValue, paramType, 1, 1);
     }
 
+    /**
+     * Whether the BSQ fee and phase is valid for a transaction.
```
Comment (ManfredKarrer) [inline core/src/main/java/bisq/core/dao/governance/param/Param.java:154]: I don't understand the meaning of that line...

File: core/src/main/java/bisq/core/dao/state/model/DaoState.java [hunk 2]
```diff
@@ -189,8 +189,7 @@ public static DaoState fromProto(PB.BsqState proto) {
                 .collect(Collectors.toMap(e -> TxOutputKey.getKeyFromString(e.getKey()), e -> TxOutput.fromProto(e.getValue())));
         Map<TxOutputKey, SpentInfo> spentInfoMap = proto.getSpentInfoMapMap().entrySet().stream()
                 .collect(Collectors.toMap(e -> TxOutputKey.getKeyFromString(e.getKey()), e -> SpentInfo.fromProto(e.getValue())));
-        Map<TxOutputKey, TxOutput> confiscatedTxOutputMap = proto.getConfiscatedTxOutputMapMap().entrySet().stream()
-                .collect(Collectors.toMap(e -> TxOutputKey.getKeyFromString(e.getKey()), e -> TxOutput.fromProto(e.getValue())));
+        List<String> confiscatedLockupTxList = proto.getConfiscatedLockupTxListList();
```
Comment (ManfredKarrer) [inline core/src/main/java/bisq/core/dao/state/model/DaoState.java:192]: Protobuffer lists are unmodifyable lists, so we need to wrap it into an arraylist

Comment (ManfredKarrer) [reply core/src/main/java/bisq/core/dao/state/model/DaoState.java:192]: I will push that change...

File: core/src/main/java/bisq/core/dao/state/DaoStateService.java [hunk 3]
```diff
@@ -679,6 +680,11 @@ public long getTotalLockupAmount() {
 
 
     // Unlock
+    public boolean isUnlockOutput(TxOutputKey key) {
```
Comment (ManfredKarrer) [inline core/src/main/java/bisq/core/dao/state/DaoStateService.java:683]: We should add unspent to the method name

File: desktop/src/main/java/bisq/desktop/main/dao/wallet/dashboard/BsqDashboardView.java [hunk 4]
```diff
@@ -106,6 +107,7 @@ public void initialize() {
         totalLockedUpAmountTextField = FormBuilder.addTopLabelReadOnlyTextField(root, gridRow, columnIndex, Res.get("dao.wallet.dashboard.totalLockedUpAmount"), Layout.FIRST_ROW_AND_GROUP_DISTANCE).second;
         totalUnlockingAmountTextField = FormBuilder.addTopLabelReadOnlyTextField(root, ++gridRow, columnIndex, Res.get("dao.wallet.dashboard.totalUnlockingAmount")).second;
         totalUnlockedAmountTextField = FormBuilder.addTopLabelReadOnlyTextField(root, ++gridRow, columnIndex, Res.get("dao.wallet.dashboard.totalUnlockedAmount")).second;
+        totalConfiscatedAmountTextField = FormBuilder.addTopLabelReadOnlyTextField(root, ++gridRow, columnIndex, Res.get("dao.wallet.dashboard.totalConfiscatedAmount")).second;
         gridRow++;
```
Comment (ManfredKarrer) [inline desktop/src/main/java/bisq/desktop/main/dao/wallet/dashboard/BsqDashboardView.java:111]: one gridRow++; need to be removed now

**Review by ManfredKarrer — APPROVED** [review]
ACK

I push a few small fixes

File: core/src/main/java/bisq/core/dao/governance/param/Param.java (hunk 1 above)
Comment (sqrrm) [reply core/src/main/java/bisq/core/dao/governance/param/Param.java:154]: That's probably a copy paste that shouldn't be included, I also don't understand it now that I read i t

File: core/src/main/java/bisq/core/dao/state/DaoStateService.java (hunk 3 above)
Comment (sqrrm) [reply core/src/main/java/bisq/core/dao/state/DaoStateService.java:683]: Agree

File: core/src/main/java/bisq/core/dao/governance/param/Param.java (hunk 1 above)
Comment (ManfredKarrer) [reply core/src/main/java/bisq/core/dao/governance/param/Param.java:154]: oki, i will remove it

**Pull request merged**